fmp_data/company/models.py

```
@@ -68,6 +77,35 @@ class CompanyProfile(BaseModel):
+            )
+            if "://" not in cleaned:
+                return f"https://{cleaned}"
+            parsed = urlparse(cleaned)
+            if not parsed.scheme or not parsed.netloc:
+                return None
+            if not parsed.hostname or "." not in parsed.hostname:
+                return None
+            adapter = TypeAdapter(AnyHttpUrl)
+            try:
+                adapter.validate_python(cleaned)
+            except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)